docs/reference/mapping/types/histogram.asciidoc

```diff
@@ -27,20 +27,6 @@ per document. Nested arrays are not supported.
 [[histogram-params]]
 ==== Parameters
```

Comment:
If there are no longer any parameters for the histogram field type I'd remove the above three lines as well. I checked and there are no links targeting the `histogram-params` ID, so it won't break anything.

Reply:
Removed this part in [fdb9735](https://github.com/elastic/elasticsearch/pull/89937/commits/fdb97358c98646a5ba39c6f5addc5d683d140ca5)